web/client-api/types/src/main/docker/theme.css

```diff
@@ -0,0 +1,18 @@
+:root {
+    --dark-color-background: #040427;
+    --dark-color-background-secondary: #0d1b37;
+    --dark-color-background-warning: #fffbef;
```

Comment:
Where are these colours coming from? This one in particular I can't see referenced anywhere else in the code base.
Pretty sure we should match the colours from other docs (defined in `root.scss`). @dsmmcken ?

Reply:
deephaven.io colors.